src/Loader/ClassLoaderInterface.php

```diff
@@ -13,5 +14,5 @@
  */
 interface ClassLoaderInterface
 {
-    public function loadClass(MapperGeneratorMetadataInterface $mapperMetadata): void;
+    public function loadClass(MapperGenerator $generator, MapperGeneratorMetadataInterface $mapperMetadata): void;
```

Comment:
I refactored this method, in order to remove the `MapperGenerator` from `EvalLoader` / `FileLoader`'s constructor.
This is really easier to refactor then, because, we don't need to create a full `MapperGenerator` [in the tests](https://github.com/nikophil/automapper/blob/feat/deffer-resolve-transformer/tests/AutoMapperBaseTest.php#L36)